src/language/templates/visitors/nmodl_visitor.cpp

```diff
@@ -52,7 +52,9 @@ using namespace ast;
 
 
 {%- macro add_child(node, child) -%}
-    {% if child.nmodl_name %}
+    {% if node.class_name == node_info.INCLUDE_NODE and child.varname == "blocks" %}
```

Comment:
Just curious what happens with the block : does extra block scope get printed ? i.e.

```
{
    INCLUDE ...
}
```
?

Reply:
No, this change is to prevent including the result of the parsed file and just print the filename. The print visitor won't add a block.